src/v2/generated/concordium.v2.rs

```diff
@@ -355,6 +355,14 @@ pub mod account_staking_info {
         /// baking committee of the current epoch.
         #[prost(message, optional, tag = "5")]
         pub pool_info:        ::core::option::Option<super::BakerPoolInfo>,
+        /// A flag indicating whether the account is currently suspended or not.
+        /// The flag has a meaning from protocol version 8 onwards. In
+        /// protocol version 8 it signals whether an account has been
+        /// suspended and is not participating in the consensus
+        /// algorithm. For protocol version < 8 the flag will always
+        /// be set to false.
+        #[prost(bool, tag = "6")]
+        pub is_suspended:     bool,
```

Comment:
This one does not seem to be reflected in the corresponding type in the SDK, unless I'm missing something?

Reply:
Good catch.